teuthology/task/install/rpm.py

```diff
@@ -99,6 +101,35 @@ def _zypper_removerepo(remote, repo_list):
             'sudo', 'zypper', '-n', 'removerepo', repo['name'],
         ])
 
+
+def _downgrade_packages(ctx, remote, pkgs, pkg_version, config):
+    """
+    Downgrade packages listed by 'downgrade_packages'
```

Comment:
Maybe add 
```
Downgrade packages which are supposed to have the same version and are provided by 'downgrade_packages' in teuthology config file. Do nothing if package list empty.
```

Reply:
as the suggested change is not quite accurate, i adapted it and revised the doc string.